examples/android/sync/app/src/androidTest/java/com/mongodb/realm/examples/java/RealmQueryTest.java

```diff
@@ -135,7 +135,7 @@ public void onSuccess(Realm realm) {
                     });
                     // :code-block-end:
                 } else {
-                    Log.e("EXAMPLE", it.getError().toString());
+                    Log.e("EXAMPLE","Failed to log in: " + it.getError().getErrorMessage());
```

Comment:
Nitpick: consistent spacing
```suggestion
                    Log.e("EXAMPLE", "Failed to log in: " + it.getError().getErrorMessage());
```